samples/colab/tflite_text_classification.ipynb

```diff
@@ -44,17 +53,44 @@
       "outputs": [],
       "source": [
         "%%capture\n",
+        "!python -m pip install --upgrade tf-nightly  # Needed for experimental_tflite_to_tosa_bytecode in TF>=2.13\n",
```

Comment:
Note that our CI does make an attempt to install common requirements, including TensorFlow, when testing notebooks... which may interfere (or at least be wasteful) with this: https://github.com/openxla/iree/blob/5aaad95ebd62121489d07504609d902fac405319/build_tools/testing/run_python_notebook.sh#L45-L51